common/src/androidTest/java/com/microsoft/identity/common/CommandDispatcherTest.java

```diff
@@ -478,6 +506,26 @@ public boolean isEligibleForEstsTelemetry() {
         }
     }
 
+    static class IErrorExceptionCommand extends BaseCommand<String> {
+        final String mErrorCode;
+
+        public IErrorExceptionCommand(@NonNull final CommandParameters parameters,
+                                @NonNull final CommandCallback callback, String errorCode) {
+            super(parameters, getTestController(), callback, "test_id");
```

Comment:
Should we pass in `errorCode` here as opposed to `test_id`?

Reply:
Yes.

Reply:
... wait, no, that's a correlation id, not an error code.